checker/src/org/checkerframework/checker/index/IndexChecker.java

```diff
@@ -3,43 +3,56 @@
 import org.checkerframework.checker.index.upperbound.UpperBoundChecker;
 
 /**
- * A type checker for preventing out-of-bounds accesses on arrays. Contains four subcheckers that do
+ * A type checker for preventing out-of-bounds accesses on arrays. Contains five subcheckers that do
  * all of the actual work, which are described here. First, the order the checkers are run in is
  * described, and then what each checker requests from the checkers that run before it is described.
  * The Index Checker itself is just an alias for the Upper Bound Checker, which runs last.
  *
  * <p>The checkers run in this order:
  *
- * <p>SameLen Checker, MinLen Checker, Lower Bound Checker, Upper Bound Checker
+ * <p>Constant Value Checker, SameLen Checker, SearchIndex Checker, Lower Bound Checker, Upper Bound
+ * Checker
+ *
+ * <p>The Constant Value Checker has no dependencies, but it does trust Positive annotations from
```

Comment:
Technically, this paragraph should be added in https://github.com/kelloggm/checker-framework/pull/155, but it's fine with me if you want to leave it here, too.